### DISCUSSION TITLE
Support for Conan Packages

### DISCUSSION THREAD

**Discussion by p0fi** [community/community#68999] [Packages]
### Select Topic Area

Product Feedback

### Body

Are there any plans to add support for Conan packages to GitHub Packages?

**Comment by esurov** [comment]
Dear @p0fi ,

As of my last knowledge, GitHub Packages primarily supports package registries for various programming languages and package managers, such as npm, Maven, RubyGems, Docker, and more. However, Conan, which is a C and C++ package manager, was not officially supported on GitHub Packages at that time.

GitHub has been known to expand its offerings and add support for additional package managers and languages over time, so it's possible that support for Conan packages may have been added since then or that there are plans to add support in the future.

To get the most up-to-date information on whether GitHub Packages supports Conan packages, I recommend visiting GitHub's official documentation or checking their blog and announcements for any updates. You can also reach out to GitHub support or their community forums for the latest information regarding the addition of new package manager support.
